scripts/gha/test_simulator.py

```diff
@@ -512,6 +512,8 @@ def _create_and_boot_simulator(apple_platform, device_name, device_os):
     # If the requested version is available, use it, otherwise default to the latest
     if (device_os not in available_versions):
       logging.warning("Unable to find version %s, will fall back to %s", device_os, available_versions[-1])
+      if FLAGS.ci:
+        print("::warning ::Unable to find version %s, will fall back to %s" % (device_os, available_versions[-1]))
```

Comment:
Since this warning is outside of context, could you add text to explain that it's regarding iOS simulator version?

Reply:
Done